db/migrate/20201007030216_add_password_confirmation_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddPasswordConfirmationToUsers < ActiveRecord::Migration[5.2]
+  def change
+    add_column :users, :password_confirmation, :string
+  end
```

Comment:
I was running into errors w/ BCrypt auth so I tried adding this password confirmation like past projects. Turned out the issue was w/ FactoryBot (was creating `password_digest` instead of `password`). I left this in here since I figured we might want it anyway for user registration? I can remove if not wanted though!

Reply:
Confirmed that it exists in wireframes.